### DISCUSSION TITLE
zh/start/download

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#788] [Announcements]
# zh/start/download

介绍 欢迎来到我们的社区！要通过机器人检查并获得Newcomer角色，请按照以下步骤进行。 1.加入我们的服务器并获得Newcomer角色 如果您还没有加入服务器，请阅读验证帖子，了解如何加入服务器并获得Newcomer角色。 2.下载必要的资源 之后，您将获得访问相应下载频道的权限：★⋅korepi-download⋅★ ::: tip 如果找不到频...

https://docs.korepi.com/zh/start/download.html

**Comment by mnbhjyghj** [comment]
6666666
